### PR TITLE
Addressing issue #99

### PR DESCRIPTION
**Pull request by vtnate**
#### What does this PR accomplish?
Use new template running method
#### How should this be manually tested?
`py.test tests/model_connectors/test_spawn_ets.py::SpawnModelConnectorSingleBuildingETSTest::test_spawn_to_modelica_and_run` - This test fails when run on my machine. @nllong can you help me? I might not have my env set up properly, or I might have made a code mistake.
#### Issue #99

### CODE REVIEW

File: tests/model_connectors/test_spawn_ets.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def test_spawn_to_modelica_and_run(self):
         mr = ModelicaRunner()
         file_to_run = os.path.abspath(
             os.path.join(
-                self.gj.scaffold.loads_path.files_dir, 'B5a6b99ec37f4de7f94020090', 'CouplingETS_SpawnBuilding.mo'
+                self.gj.scaffold.loads_path.files_dir, 'B5a6b99ec37f4de7f94020090', 'building.mo'
```
Comment (nllong) [inline tests/model_connectors/test_spawn_ets.py:83]: I think we have to keep the file name the same to keep Modelica happy. Use  CouplingETS_SpawnBuilding.mo.

File: geojson_modelica_translator/model_connectors/spawnBui_ETS_Coupling.py [hunk 2]
```diff
@@ -215,14 +215,23 @@ def to_modelica(self, scaffold):
...
+                #     data=template_data,
+                # )
+                # with open(os.path.join(os.path.join(b_modelica_path.files_dir, "CouplingETS_SpawnBuilding.mo")),
+                #           "w") as f:
+                #     f.write(file_data)
+
+                self.run_template(
+                    spawn_ets_coupling_template,
+                    # time_series_building_template,
+                    os.path.join(b_modelica_path.files_dir, "building.mo"),
```
Comment (nllong) [inline geojson_modelica_translator/model_connectors/spawnBui_ETS_Coupling.py:230]: I think we have to keep the file name the same to keep Modelica happy. Use  CouplingETS_SpawnBuilding.mo.

Comment (vtnate) [reply geojson_modelica_translator/model_connectors/spawnBui_ETS_Coupling.py]: Doh, of course. Thanks.

**Comment by vtnate** [comment]
@nllong I updated the files you requested with the new method. There are other instances of the existing call that look similar in `ets_template.py` and `time_series_ets_coupling.py`. I looked at replacing those too, but broke tests.

**Pull request merged**